taipy/core/_scheduler/_dispatcher/_standalone_job_dispatcher.py

```diff
@@ -39,11 +41,14 @@ def _dispatch(self, job: Job):
         future = self._executor.submit(
             self._run_wrapped_function, Config.global_config.storage_folder, job.id, job.task
         )
+        _set_process_in_scheduler(job.id, future)
```

Comment:
From my understanding of the problem, set_process must be called before add_done_callback because some of the job is too fast, after it is done, the call back (update_status -> pop_process) will be called before the set_process was called